tests/StackExchange.Redis.Tests/Sets.cs

```diff
@@ -14,234 +14,244 @@ public class Sets : TestBase
-
-                Assert.Equal(10, db.SetLength(key));
+            db.KeyDelete(key, CommandFlags.FireAndForget);
+            for (int i = 1; i < 11; i++)
+            {
+                db.SetAdd(key, i, CommandFlags.FireAndForget);
             }
+
+            var t = db.SetPopAsync(key, count: 0);
+            Assert.True(t.IsCompleted); // sync
+            var arr = await t;
+            Assert.Empty(arr);
+            Assert.Equal(10, db.SetLength(key));
         }
 
         [Fact]
         public void SetAdd_Zero()
         {
-            using (var conn = Create())
-            {
-                var db = conn.GetDatabase();
-                var key = Me();
+            using var conn = Create();
+            var db = conn.GetDatabase();
+            var key = Me();
 
-                db.KeyDelete(key, CommandFlags.FireAndForget);
+            db.KeyDelete(key, CommandFlags.FireAndForget);
 
-                var result = db.SetAdd(key, Array.Empty<RedisValue>());
-                Assert.Equal(0, result);
+            var result = db.SetAdd(key, Array.Empty<RedisValue>());
+            Assert.Equal(0, result);
 
-                Assert.Equal(0, db.SetLength(key));
-            }
+            Assert.Equal(0, db.SetLength(key));
         }
 
         [Fact]
         public async Task SetAdd_Zero_Async()
         {
-            using (var conn = Create())
-            {
-                var db = conn.GetDatabase();
-                var key = Me();
-
-                db.KeyDelete(key, CommandFlags.FireAndForget);
+            using var conn = Create();
+            var db = conn.GetDatabase();
+            var key = Me();
 
-                var t = db.SetAddAsync(key, Array.Empty<RedisValue>());
-                Assert.True(t.IsCompleted); // sync
-                var count = await t;
-                Assert.Equal(0, count);
+            db.KeyDelete(key, CommandFlags.FireAndForget);
 
-                Assert.Equal(0, db.SetLength(key));
-            }
+            var t = db.SetAddAsync(key, Array.Empty<RedisValue>());
+            Assert.True(t.IsCompleted); // sync
+            var count = await t;
+            Assert.Equal(0, count);
+            Assert.Equal(0, db.SetLength(key));
         }
 
         [Fact]
         public void SetPopMulti_Nil()
         {
-            using (var conn = Create())
-            {
-                Skip.IfBelow(conn, RedisFeatures.v3_2_0);
+            using var conn = Create();
+            Skip.IfBelow(conn, RedisFeatures.v3_2_0);
 
-                var db = conn.GetDatabase();
-                var key = Me();
+            var db = conn.GetDatabase();
+            var key = Me();
 
-                db.KeyDelete(key, CommandFlags.FireAndForget);
+            db.KeyDelete(key, CommandFlags.FireAndForget);
 
-                var arr = db.SetPop(key, 1);
-                Assert.Empty(arr);
-            }
+            var arr = db.SetPop(key, 1);
+            Assert.Empty(arr);
```

Comment:
Let's please only use the new syntax in the PRs (doing as little diff as possible) - I'll do a separate cleanup pass that's much cleaner overall git history. If we do these mass cleanups mixed in other PRs, history is very confusing later on.